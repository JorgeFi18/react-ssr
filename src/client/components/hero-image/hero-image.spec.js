import React from 'react';
import renderer from 'react-test-renderer';
import HeroImage from './hero-image.component';

describe('Hero Image component', () => {

    it('Should match with snapshot', () => {
        const component = renderer.create(
            <HeroImage thumbnail='path/to/my-image' />
        )

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});