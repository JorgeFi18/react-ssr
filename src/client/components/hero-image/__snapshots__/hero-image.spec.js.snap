// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Image component Should match with snapshot 1`] = `
<img
  alt="hero"
  className="hero-card__image"
  src="path/to/my-image"
/>
`;
